mvrx/src/main/kotlin/com/airbnb/mvrx/MavericksViewModel.kt

```diff
@@ -440,33 +434,42 @@ abstract class MavericksViewModel<S : MavericksState>(
         }
     }
 
+    private fun <T : Any> Flow<T>.configureAndFilter(lifecycleOwner: LifecycleOwner, subscriptionId: String, skipCount: Int = 0): Flow<T> {
```

Comment:
I think it would be nice to expose a Flow extension function with similar behavior so users can have easy delivery mode configuration when working with the `stateFlow` directly. We'd probably want to think a bit more carefully about what that Api looks like though